shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_component_impl.cpp

```diff
@@ -89,12 +108,18 @@ ErrCode TmsClientComponentBaseImpl<Impl>::getDescription(IString** description)
 {
     OPENDAQ_PARAM_NOT_NULL(description);
 
-    return daqTry([&]
+    try
     {
         StringPtr descObj = this->client->readDescription(this->nodeId);
         *description = descObj.detach();
```

Comment:
Same comment as the `active` getter. The same applies for all other getters (I won't add a comment to each instance).